bot/resources/tags/on-message-event.md

```diff
@@ -0,0 +1,16 @@
+---
+embed:
+    title: "The discord.py `on_message` event"
+---
+
+Registering the `on_message` event with [`@bot.event`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.event) will override the default behavior of the event. This may cause prefix commands to stop working, because they rely on the default `on_message` event handler.
+
+Instead, use [`@bot.listen`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.listen) to add a listener. Listeners get added alongside the default `on_message` event, rather than overriding it, so prefix commands can still be invoked as usual:
+```python
+@bot.listen('on_message')
+async def message_listener(message):
```

Comment:
Perhaps we can recommend the traditional usage of `@bot.listen()` where the function name is used as the event:
```suggestion
@bot.listen()
async def on_message(message):
```

Reply:
I would provide examples of both implementation methods. 
```suggestion
# Method 1
@bot.listen('on_message')
async def message_listener(message):
    ...  # do stuff here

# Method 2
@bot.listen()
async def on_message(message):
```